analyzers/src/SonarAnalyzer.CFG/LiveVariableAnalysis/RoslynLiveVariableAnalysis.cs

```diff
@@ -78,25 +78,21 @@ protected override State ProcessBlock(BasicBlock block)
 
     private void ResolveCaptures()
     {
-        foreach (var operation in Cfg.Blocks
+        foreach (var flowCapture in Cfg.Blocks
                                     .Where(x => x.EnclosingRegion.EnclosingRegionOrSelf(ControlFlowRegionKind.LocalLifetime) is not null)
                                     .SelectMany(x => x.OperationsAndBranchValue)
-                                    .ToExecutionOrder())
+                                    .ToExecutionOrder()
+                                    .Select(x => x.Instance.AsFlowCapture())
+                                    .OfType<IFlowCaptureOperationWrapper>())
```

Comment:
The `OfType` is a hidden null check that doesn't remove the nullability problem. Make it `.Where(x => x.Instance.IsKind(..)` and `.Select(x => x.ToFlowCapture())`